packages/@lwc/engine/src/framework/attributes.ts

```diff
@@ -99,26 +91,38 @@ export const globalHTMLProperties: {
     hidden: {
         attribute: 'hidden',
     },
-    itemScope: {
```

Comment:
Removed all the `item*` microdata properties because they only exist as attributes:
```
None of these content attributes have corresponding IDL attributes:
https://html.spec.whatwg.org/multipage/microdata.html
```